setup.cfg

```
@@ -37,8 +37,7 @@ install_requires =
     jupyter_core
     jupyterlab_server~=2.10
     jupyter_server>=1.16.0,<3
-    jupyter_ydoc~=0.2.2
-    jupyter_server_ydoc>=0.3.0,<0.4.0
+    jupyter_server_ydoc>=0.4.0,<0.5.0,
```

Looks like there is an extra comma:

```suggestion
    jupyter_server_ydoc>=0.4.0,<0.5.0
```

---

Thanks @jtpio!